### DISCUSSION TITLE
Please tell me your favorite PC game.

### DISCUSSION THREAD

**Discussion by NM-KIR** [NAIST-SE/AttractingContributors#915] [2023 Attraction Contributions]
Please add your favorite PC game to the list!
(好きなPCゲームをリストに追加してください！)

https://github.com/NM-KIR/Your-hobby

**Comment by rfLENtlr** [comment]
I lile Apex Legends!
I sent a pull request.
Would you like to contribute to my repository as well?
https://github.com/rfLENtlr/AttractingContributors-OS

**Comment by bconfessor** [comment]
I like Fortnite, I sent you a pull request! 
Could you please tell me your favorite song?? 
https://github.com/bconfessor/FavoriteSong/blob/main/README.md

**Reply by Gloria109** [reply]
I like Fortnite too
